### DISCUSSION TITLE
Wyszukiwanie zamówień po numerze oferty

### DISCUSSION THREAD

**Discussion by bswierzewski** [allegro/allegro-api#8426] [Suggestions - rejected]
Witam serdecznie, już kilkukrotnie spotkałem się z problemem połączenia oferty z zamówieniami i aktualnie nie istnieje sposób by pobrać wszystkie zamówienia złożone na podstawie danej oferty. 

Sugestia: Dla zasobu /order/checkout-forms dodać parametr wyszukiwania zawierający pole offer.id który pozwoli filtrować zamówienia po ofercie.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
